common/scaffolding/typography.html

```diff
@@ -0,0 +1,45 @@
+---
+{
+	"title": "Typography",
+	"language": "en",
+	"altLangPage": "typographie.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2024-03-06"
+}
+---
+
+<p>The following content is using HTML semantic headings for demonstrating their rendering and styling. An aria role paragraph was added to them to minimize the impact with assistive technology.</p>
+
+<h2>Headings</h2>
+
+<h1 role="paragraph">This is an h1 heading element</h1>
+<h2 role="paragraph">This is an h2 heading element</h2>
+<h3 role="paragraph">This is an h3 heading element</h3>
+<h4 role="paragraph">This is an h4 heading element</h4>
+<h5 role="paragraph">This is an h5 heading element</h5>
+<h6 role="paragraph">This is an h6 heading element</h6>
+<pre><code>
+&lt;h1&gt;This is an h1 heading element &lt;/h1&gt;
+&lt;h2&gt;This is an h2 heading element &lt;/h2&gt;
+&lt;h3&gt;This is an h3 heading element &lt;/h3&gt;
+&lt;h4&gt;This is an h4 heading element &lt;/h4&gt;
+&lt;h5&gt;This is an h5 heading element &lt;/h5&gt;
+&lt;h6&gt;This is an h6 heading element &lt;/h6&gt;
+</code></pre>
+
+<h2>Paragraph</h2>
+
+<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+<pre><code>
+&lt;p&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.&lt;/p&gt;
+</code></pre>
+
+<h2>Limited width content</h2>
+
+<p>For an example of content with limited width, visit the following link: <a href="https://wet-boew.github.io/GCWeb/templates/content-limit-en.html">Content page - Limited width content</a></p>
+
+<h2>Text level semantics</h2>
+
+<p>For an example of text-level semantics which showcases links and grouping related elements, visit the following link: <a href="https://wet-boew.github.io/GCWeb/common/text-level-semantics/text-level-semantics.html">Text level semantics</a></p>
```

Comment:
Fix the broken link and use relative link instead of absolute URLs.

Reply:
Fixed and addressed in commit "Fixed links and removed breadcrumbs".